examples/aws_cli.py

```diff
@@ -117,7 +121,7 @@ def StartAnalysisVm(args: 'argparse.Namespace') -> None:
     args (argparse.Namespace): Arguments from ArgumentParser.
   """
   if args.attach_volumes and len(args.attach_volumes.split(',')) > 11:
-    print('error: --attach_volumes must be < 11')
+    logger.error('error: --attach_volumes must be < 11')
```

Comment:
wouldn't be there an Error automatically printed by the logger before the message, shouldn't the 'error:' be removed, it looks to me like it will be: ERROR  error: --atta...